apps/linode-marketplace-filecloud/roles/solr/tasks/solr-config.yml

```diff
@@ -0,0 +1,63 @@
+- name: checking if fccore is deleted
+  file: 
+    path: /opt/solrfcdata/var/solr/data/fccore
+    state: absent
+
+- name: copying fcskel into dest
+  copy:
+    src: /var/www/html/thirdparty/overrides/solarium/Solarium/fcskel
+    dest: /opt/solrfcdata/var/solr/data/
+
+- name: renaming fcskel to fccore
+  shell:
+    cmd: mv /opt/solrfcdata/var/solr/data/fcskel /opt/solrfcdata/var/solr/data/fccore
+
+- name: changing owner of fccore
+  file:
+    path: /opt/solrfcdata/var/solr/data/fccore
+    owner: solr
+    state: directory
+    recurse: yes
+
+- name: grabbing private ip
+  shell: 
+    cmd: hostname -i
+  register: private_ip
```

Comment:
`private_ip` should be a variable declared in `group_vars/linode/vars` file. If we are going to be calling `private_ip`,  it must be a variable.